src/components/Calculator.js

```diff
@@ -17,34 +17,51 @@ function Calculator() {
     ? `${state.total || ''} ${state.operation || ''} ${state.next}`
     : state.total || '0';
 
+  return (
+    <section>
+
+      <div>
+        <div className="nav">
+          <p className="display-board">{displayValue}</p>
+        </div>
+        <div className="buttons">
+          <button type="button" onClick={handleClick} className="btn  function">AC</button>
+          <button type="button" onClick={handleClick} className="btn  function">+/-</button>
+          <button type="button" onClick={handleClick} className="btn  function">%</button>
+          <button type="button" onClick={handleClick} className="btn operator">+</button>
+          <button type="button" onClick={handleClick} className="btn number">7</button>
+          <button type="button" onClick={handleClick} className="btn number">8</button>
+          <button type="button" onClick={handleClick} className="btn number">9</button>
+          <button type="button" onClick={handleClick} className="btn operator">-</button>
+          <button type="button" onClick={handleClick} className="btn number">4</button>
+          <button type="button" onClick={handleClick} className="btn number">5</button>
+          <button type="button" onClick={handleClick} className="btn number">6</button>
+          <button type="button" onClick={handleClick} className="btn operator">*</button>
+          <button type="button" onClick={handleClick} className="btn number">1</button>
+          <button type="button" onClick={handleClick} className="btn number">2</button>
+          <button type="button" onClick={handleClick} className="btn number">3</button>
+          <button type="button" onClick={handleClick} className="btn operator">/</button>
```

Comment:
- [ ] The `*` and `/` operations don’t work they give an error, and that is because in the `operate.js` file the multiplication sign is `x` and the division sign is `÷` but you used other signs `*` and `/`. Kindly fix that 🙃 🙃 

<img width="982" alt="Screenshot 2023-06-21 at 15 49 01" src="https://github.com/danielochuba/Math-magicians/assets/74501523/fd7c29f7-5c76-4a43-b5ad-319e29c0a460">

<img width="982" alt="Screenshot 2023-06-21 at 15 49 17" src="https://github.com/danielochuba/Math-magicians/assets/74501523/1b5af116-9fa8-4cff-9352-6fcd49c1fa70">

Reply:
@cynthiainga thank you for pointing out the errors